### PR TITLE
feat : added alert and avatar components

### PR DESCRIPTION
**Pull request by Priya-Patel31**

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Looks great !

**Review by JantuDeb — APPROVED** [review]
Clean and awesome code.

**Review by ultronjs — APPROVED** [review]
Looks good but code snippets is looking different at different places. For Eg-in alert some has a background and some don't.

**Review by sks-21 — APPROVED** [review]
Nice Code structure. Looks good to merge.

File: component/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    padding: 1rem 0.5rem;
+    margin-bottom: 1rem;
+    border-radius: 0.3rem;
+  }
+  .alert-info {
+    border: 2px solid var(--dark-blue);
+    background-color: var(--sky-blue);
+    color: var(--dark-blue);
+    display: flex;
+    justify-content: space-between;
```
Comment (Ankur9669) [inline component/alert/alert.css:37]: Please remove display:flex as it is unused.

File: component/alert/alert.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  You can copy html part from below code snippet and use it where needed.
+</p>
+
+<div class="alerts-list-container outline">
+  <div class="alert-common alert-info">
+    A info alert example with a text link!!
+  </div>
+  <div class="alert-common alert-secondary">A secondary alert example!!</div>
+  <div class="alert-common alert-success">
+    <span><i class="fas fa-check-circle"></i></span>A success alert example!!
```
Comment (Ankur9669) [inline component/alert/alert.html]: You can remove span from here

File: component/avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    border-radius: 50%;
+    margin-left: 1rem;
+  }
+  .text-avatar {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+  }
+
```
Comment (Ankur9669) [inline component/avatar/avatar.css]: Text avatar is unused css, you can remove it

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
Please Do these changes. By the way the code is great!!.

File: component/alert/alert.css (hunk 1 above)
Comment (Priya-Patel31) [reply component/alert/alert.css:37]: done!!

File: component/alert/alert.html (hunk 2 above)
Comment (Priya-Patel31) [reply component/alert/alert.html]: removed!!

**Review by Ankur9669 — APPROVED** [review]
The code looks great to me. Go ahead and merge it👍

**Pull request merged**